test/system/065-cp.bats

```diff
@@ -68,6 +72,38 @@ load helpers
        "copy into nonexistent path in container"
 
     run_podman rm -f cpcontainer
+
+    # CREATED container
+    while read id dest dest_fullname description; do
+        run_podman create --name cpcontainer --workdir=/srv $cpimage cat $dest_fullname
+        run_podman cp $srcdir/hostfile$id cpcontainer:$dest
+        run_podman start -a cpcontainer
+        is "$output" "${randomcontent[$id]}" "$description (cp -> ctr:$dest)"
+        run_podman rm -f cpcontainer
+    done < <(parse_table "$tests")
+
+    run_podman rmi -f $cpimage
+}
+
+@test "podman cp file from host to container tmpfs mount" {
+    srcdir=$PODMAN_TMPDIR/cp-test-file-host-to-ctr
+    mkdir -p $srcdir
+    content=$(random_string 20)
```

Comment:
I've developed a new habit when writing tests like this: adding placeholder markers to random content.
```bash
   content=tmpfile-content-$(random_string 20)
```
The main benefit is, when there's a test failure, the `tmpfile-content` text makes it very easy for a maintainer to see the error in better context: to understand what the expectation is and where it's set. Also importantly, when reading a long error log, it makes it easier to see which random string is which, and ignore the red herrings.

Reply:
I'll add that. That can be super helpful!